src/discussions/post-comments/comments/CommentsView.jsx

```diff
@@ -92,7 +92,7 @@ const CommentsView = ({ threadType }) => {
              variant="plain"
              block="true"
              className="card mb-4 px-0 border-0 py-10px mt-2 font-style font-weight-500
-                    line-height-24 text-primary-500"
+                    line-height-24 text-primary-500 btn-plain"
```

Comment:
I don't think this is needed. When I looked at this before I found

> Setting the variant to `plain` adds the css `btn-plain` class to the `Button`

and this button is already being set to `variant="plain"`

Reply:
right, fixed